solidity/test/Consumer_test.js

```diff
@@ -98,9 +98,7 @@ contract('Consumer', () => {
       })
 
       it('does not accept the data provided', async () => {
-        await assertActionThrows(async () => {
-          await oc.fulfillData(otherId, response, {from: oracleNode})
-        })
+        await oc.fulfillData(otherId, response, {from: oracleNode})
```

Comment:
If this no longer throws and subsequently revert, can we change the copy in the `it`?

Reply:
It still seems accurate to me. What would you change it to?

Reply:
It still transfers link?

Reply:
Doesn't technically call `transfer`, but it unlocks the deposit allowing for the operator to withdraw it.